### PR TITLE
[14.0] fleet_vehicle_inspection

### PR DESCRIPTION
**Pull request by mariadforgeflow**
* Add Spanish Translations
* Fix Access Error

### CODE REVIEW

File: fleet_vehicle_inspection/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,754 @@
...
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__odometer_unit
+msgid "Odometer Unit"
+msgstr "Unidad del Cuentakilómetros"
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,help:fleet_vehicle_inspection.field_fleet_vehicle_inspection__odometer
+#: model:ir.model.fields,help:fleet_vehicle_inspection.field_fleet_vehicle_inspection__odometer_id
+    msgid "Odometer measure of the vehicle at the moment of this log"
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgid "Odometer measure of the vehicle at the moment of this log"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,754 @@
...
+#: model:ir.model,name:fleet_vehicle_inspection.model_fleet_vehicle_inspection_line_image
+msgid "Fleet Vehicle Inspection Line Image"
+msgstr "Imagen de Línea de Inspección de Vehículos de Flota"
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__message_follower_ids
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__message_follower_ids
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__message_follower_ids
+msgid "Followers"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Seguidores"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,754 @@
...
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__message_channel_ids
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__message_channel_ids
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__message_channel_ids
+msgid "Followers (Channels)"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Seguidores (Canales)"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 4]
```diff
@@ -0,0 +1,754 @@
...
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__message_partner_ids
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__message_partner_ids
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__message_partner_ids
+msgid "Followers (Partners)"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Seguidores (Empresas)"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 5]
```diff
@@ -0,0 +1,754 @@
...
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,help:fleet_vehicle_inspection.field_fleet_vehicle_inspection__activity_type_icon
+#: model:ir.model.fields,help:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__activity_type_icon
+#: model:ir.model.fields,help:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__activity_type_icon
+msgid "Font awesome icon e.g. fa-tasks"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Icono de Font Awesome ej. fa-tasks"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 6]
```diff
@@ -0,0 +1,754 @@
...
+#: model:ir.model.fields.selection,name:fleet_vehicle_inspection.selection__fleet_vehicle_inspection__direction__in
+msgid "IN"
+msgstr "ENTRADA"
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__activity_exception_icon
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__activity_exception_icon
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__activity_exception_icon
+msgid "Icon"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Icono"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 7]
```diff
@@ -0,0 +1,754 @@
...
+#: model_terms:ir.ui.view,arch_db:fleet_vehicle_inspection.fleet_vehicle_inspection_line_form_view
+msgid "Instruction"
+msgstr "Instrucción"
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__message_is_follower
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__message_is_follower
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__message_is_follower
+msgid "Is Follower"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Es un seguidor"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 8]
```diff
@@ -0,0 +1,754 @@
...
+msgid "Last Updated by"
+msgstr ""
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__write_date
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__write_date
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__write_date
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line_image__write_date
+msgid "Last Updated on"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Última actualización por"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 9]
```diff
@@ -0,0 +1,754 @@
...
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__message_main_attachment_id
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__message_main_attachment_id
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__message_main_attachment_id
+msgid "Main Attachment"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Adjuntos principales"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 10]
```diff
@@ -0,0 +1,754 @@
...
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__message_has_error
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__message_has_error
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__message_has_error
+msgid "Message Delivery error"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Error de Envío de Mensaje"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 11]
```diff
@@ -0,0 +1,754 @@
...
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection__message_ids
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__message_ids
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__message_ids
+msgid "Messages"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Mensajes"
```

File: fleet_vehicle_inspection/i18n/es.po [hunk 12]
```diff
@@ -0,0 +1,754 @@
...
+#: model_terms:ir.ui.view,arch_db:fleet_vehicle_inspection.fleet_vehicle_inspection_line_form_view
+msgid "Todo"
+msgstr ""
+
+#. module: fleet_vehicle_inspection
+#: model:ir.model.fields,help:fleet_vehicle_inspection.field_fleet_vehicle_inspection__activity_exception_decoration
+#: model:ir.model.fields,help:fleet_vehicle_inspection.field_fleet_vehicle_inspection_item__activity_exception_decoration
+#: model:ir.model.fields,help:fleet_vehicle_inspection.field_fleet_vehicle_inspection_line__activity_exception_decoration
+msgid "Type of the exception activity on record."
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline fleet_vehicle_inspection/i18n/es.po]:
```suggestion
msgstr "Tipo de actividad de excepción registrada."
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by JoanMForgeFlow — APPROVED** [review]
LGTM!

**Comment by MiquelRForgeFlow** [comment]
@ivantodorovich

**Review by marcelsavegnago — APPROVED** [review]

**Comment by marcelsavegnago** [comment]
@mariadforgeflow Shouldn't the Spanish translation be done through the weblate?

**Comment by ivantodorovich** [comment]
> @mariadforgeflow Shouldn't the Spanish translation be done through the weblate?

Indeed translations should be done in weblate: https://translation.odoo-community.org/projects/fleet-14-0/fleet-14-0-fleet_vehicle_inspection/

However here since it's already done, and it's a whole new language file (no risk of de-syncronization) I say we can merge it

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**